.github/workflows/docs.yml

```diff
@@ -8,32 +8,32 @@ jobs:
   generate-docs:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-        with:
-          ref: "gh-pages"
+    - uses: actions/checkout@v3
+      with:
+        ref: gh-pages
 
-      - name: Create conda environment
-        run: conda create --quiet -c conda-forge --name shpc spython
+    - name: Create conda environment
+      run: conda create --quiet -c conda-forge --name shpc spython
 
-      - name: Install shpc
-        run: |
-          export PATH="/usr/share/miniconda/bin:$PATH"
-          root=$PWD
-          source activate shpc
-          cd /tmp
-          git clone https://github.com/singularityhub/singularity-hpc
-          cd singularity-hpc
-          pip install -e .
-          cd $root
+    - name: Install shpc
+      run: |
+        export PATH="/usr/share/miniconda/bin:$PATH"
+        root=$PWD
+        source activate shpc
+        cd /tmp
+        git clone https://github.com/singularityhub/singularity-hpc
+        cd singularity-hpc
+        pip install -e .
+        cd $root
 
-      - name: Generate docs
-        run: |
-          export PATH="/usr/share/miniconda/bin:$PATH"
-          source activate shpc
-          /bin/bash generate.sh
+    - name: Generate docs
+      run: |
+        export PATH="/usr/share/miniconda/bin:$PATH"
+        source activate shpc
+        /bin/bash generate.sh
 
-      - name: Deploy 🚀
-        uses: JamesIves/github-pages-deploy-action@4.1.1
-        with:
-          branch: gh-pages
-          folder: .
+    - name: Deploy 🚀
+      uses: JamesIves/github-pages-deploy-action@ba1486788b0490a235422264426c45848eac35c6 # v4.4.1
```

Comment:
Just curious: why did it convert some version numbers to checksums ?

Reply:
Nevermind, I found the answer at https://github.com/vsoch/action-updater#version (RTFM, I suppose ...)

Reply:
That’s the default for untrusted repos, to use a commit. There is currently a short list in settings (e.g GitHub maintained ones) that we use major tags for from releases.

Reply:
haha, I'm happy to answer your questions directly @muffato ! Those docs aren't complete or really nice looking yet :)

Reply:
👍🏼